src/parse-sfc.js

```diff
@@ -22,7 +22,7 @@ module.exports = function parse(source) {
 function detectIndent(sfcBlock, source) {
   const nonEmptyPaddingsPerLine = source.substring(sfcBlock.start, sfcBlock.end)
     .split('\n')
-    .filter(line => line !== '')
+    .filter(line => !line.match(/^\s*$/))
```

Comment:
This was causing the entire block to get indented one extra level when there were whitespace-only lines or mixed tabs.

Now it matches that beginning/end empty lines, but also ignores those other weird cases.